### ISSUE TITLE
Mockup : Projects Page

### ISSUE THREAD

**Issue by LinnierGames** [SRJC-Computer-Science-Club/CS-Website#3]
scroll down to the -> [outline](#outline) <- or [resources](#res)

<a href="https://github.com/SRJC-Computer-Science-Club/cs-website/files/383108/res.zip"><img src="https://cloud.githubusercontent.com/assets/1758210/17081828/b3017ac6-511c-11e6-96a0-a19488686729.png"></a>

This one may require some java script since the **header bar** under the nav bar will freeze when scrolling down ( shown in the preview video in the res folder ). Other than that, a horizontal scroll view to page through upcoming events, if any, of the projects.

The **Top Projects** cell is more information than the homepage on #2. The project cell will have an image, status, number of members and a cell for each member, a title and a link to the git hub issues showing the number of tasks left, a placeholder text for the number of positions opened ( after #7 is finished then this will be worked on to pull realtime data ), lastly the most recent project post which will be a placeholder text since this will be a table of project posts.

Next, are the **Secondary Projects** cell. This only shows less information but a project is a project in the database. An image will show with the status of the project, number of members, a fraction of how many tasks are done and how much in total, and of course the title of the project.

Last are **Archived and Completed Projects** cell. Only the title of the project is shown with the status title ( e.g. Completed, Revoked, etc. ), and the number of members in the project.

Lots of placeholder text but it's something that needs to be done :)

<h3 id="outline">Issue's Outline</h3>

- [ ] Next Events
- [x] Top Project Cell
- [x] Secondary Project Cell
- [x] Archived and Completed Project Cell

<h3 id="res">Resources</h3>

- raw mockup file ![desktop](https://cloud.githubusercontent.com/assets/1758210/17081919/45edbf5a-511f-11e6-8aa3-366bf78db15e.png)
- preview of the scrolling behavior of the header cell
- pdf of the raw mockup

**Comment by LinnierGames** [comment]
here is a preview of the webpage

![screenshot](https://cloud.githubusercontent.com/assets/1758210/17314445/574a74ae-5819-11e6-9e43-c9daee9150c7.png)

**Comment by LinnierGames** [comment]
the footer section is not part of this issue :P instead #6
